tests/pds/common.go

```diff
@@ -162,3 +167,84 @@ func RunWorkloads(params pdslib.WorkloadGenerationParams, ds PDSDataService, dep
 	return pod, dep, err
 
 }
+
+// Check the DS related PV usage and resize in case of 90% full
+func CheckPVCtoFullCondition(context []*scheduler.Context) error {
+	log.Infof("Start polling for the pvc consumption by the DS")
```

Comment:
nit: update the logger like "check pvc usage"

Reply:
Done !